src/Concordium/Client/Output.hs

```diff
@@ -1160,6 +1166,9 @@ showRatio r =
       den = denominator r
    in show num ++ " / " ++ show den ++ " (approx " ++ show (realToFrac r :: Double) ++ ")"
 
+showPartsPerHundredThousands :: Types.PartsPerHundredThousands -> String
+showPartsPerHundredThousands (Types.PartsPerHundredThousands val) = show val ++ "/" ++ "100000"
```

Comment:
I think it might be better to render this as a percentage.

Reply:
I think just using the Show instance of PartsPerHundredThousands is also fine.